ui-participant/src/participant/EditParticipantProfileModals.tsx

```diff
@@ -359,6 +361,24 @@ export function EditMailingAddressModal(props: EditModalProps) {
     }
   }
 
+  const tryValidateAndSave = async () => {
+    // only validate US addresses
+    if (mailingAddress.country === 'US') {
+      try {
+        await validateAndSave()
+      } catch (e) {
+        await Api.log({
```

Comment:
no reason to `await` this call -- generally it's a bad idea to `await` logging

Reply:
also, would this `validateAndSave()` ever fail for client-side reasons?  I'm wondering if we need this extra log at all, or whether the server-side logging is sufficient

Reply:
You're probably right - I can't quite remember why I did this Api.log so I'll remove it